libs/portals/core/src/types/portalCore.ts

```diff
@@ -57,6 +57,21 @@ export interface PortalNavigationItem {
    * Active state if current path is exact match
    */
   activeIfExact?: boolean
+
+  /**
+   * Service provider
```

Comment:
~ This comment is not providing any extra context as it just repeating the variable name. It would be nice to add a bit more detail for what this property is used.